android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java

```diff
@@ -5,6 +5,8 @@
 import android.net.Uri;
 import android.support.design.widget.Snackbar;
 import android.support.graphics.drawable.VectorDrawableCompat;
+import android.support.v4.content.ContextCompat;
+import android.support.v4.graphics.drawable.DrawableCompat;
```

Comment:
To avoid such errors do a Code Inspection before each PR. You could remove such errors easily.